test/Parse/ConditionalCompilation/basicIdentity.swift

```diff
@@ -27,7 +27,7 @@ var i = Foo()
 #endif
 var j = i
 
-#if os(OSX)
+#if os(macOS)
```

Comment:
Be careful changing the ConditionalCompilation tests.  I would like this one preserved and a `macOS` test added here.

Reply:
Good point! I think the both `OSX` and `macOS` conditions are specifically tested in https://github.com/apple/swift/blob/c26f40b45b3d21fb5844c71549ee23baa4f7b855/test/Parse/ConditionalCompilation/x64OSXTarget.swift#L4-L15 (https://github.com/apple/swift/pull/3395/files#diff-ddecd035914171df461e408178c90007),

but preserving the `OSX` test and adding a `macOS` test to this file would be good anyway.